include/glow/Graph/Graph.h

```diff
@@ -1081,6 +1082,9 @@ Function *differentiate(Function *F, const TrainingConfig &config,
                         llvm::StringRef newFuncName = "",
                         VariableGradientsList *varGrads = nullptr);
 
+/// If \p PH is an output placeholder, \returns the SaveNode.
```

Comment:
A better message:
/// \returns the first SAveNode user of the placeholder \p PH or nullptr if none are found.